spec/support/test_environment.rb

```diff
@@ -6,6 +6,7 @@
   config.include RspecTagMatchers
   config.include CustomMacros
   config.mock_with :rspec
+  config.deprecation_stream = '/dev/null'
```

Comment:
Ok, let's get rid of this and maybe someone sometime will fix them. I expect a lot can be done with regular expressions and find & replace.